csrc/ops/alias.cpp

```diff
@@ -148,13 +149,15 @@ TensorView* reshape(TensorView* inp_tv, const std::vector<Val*>& new_sizes) {
       }
       new_size = div(numel, other_new_numel);
       new_size = simplifyExpr(new_size);
+    } else if (new_size->isConstScalar() && new_size->evaluate() == 1) {
+      iter_type = IterType::Broadcast;
     }
     if (new_size->dtype() != DataType::Index) {
       new_size = castOp(DataType::Index, new_size);
     }
     auto rf_id =
         IterDomainBuilder(FusionGuard::getCurFusion()->zeroVal(), new_size)
-            .iter_type(IterType::Symbolic)
+            .iter_type(iter_type)
```

Comment:
I'm now trying to mark the reshape output's iterdomain as broadcast when we see a static size-1 here.
But this triggered an assert in `ValidateDomainEquivalence`...

I'll try to dive into the call stack to figure out what else I missed.